src/styles/pages/main/main.scss

```diff
@@ -0,0 +1,377 @@
+        }
+    }
+
+    &__btn {
+        background-color: black;
+        border: 1px solid white;
+        border-radius: 10px;
+
+        &:hover {
+            background-color: grey;
+        }
+
+        button {
+            color: white;
+            font-size: 20px;
+            width: 90px;
+            padding: 10px;
+        }
+    }
+}
+
+.banner {
+    height: 100vh;
+
+    .background-image {
+        position: absolute;
+        inset: 0;
+        width: 100%;
+        height: 100%;
+        background-position: center;
+        background-size: cover;
+        background-repeat: no-repeat;
+        z-index: -1;
+        top: 58px;
+    
+        &::before {
+            content: "";
+            width: 100%;
+            height: 100%;
+            position: absolute;
+        }
+    
+        &_main   {
+            background-image: url(https://kartinki.pics/uploads/posts/2021-04/1617313820_4-p-foni-dlya-menyu-5.jpg) !important;
+        }
+    }
+    
+    .major {
+
+        .container {
+            max-width: 1170px;
+            margin: 0 auto;
+            padding: 270px 20px;
+        }
+    
+        .major_heading {
+            text-align: center;
+            color: white;
+    
+            h3 {
+                color: white;
+                font-size: 50px;
+            }
+    
+            h1 {
+                color: white;
+                font-size: 100px;
+                padding: 30px 0;
+                font-family: 'Dancing Script', cursive;
+            }
+    
+            h2 {
+                color: white;
+                font-size: 30px;
+            }
+        }
+    }
+}
+
+.history {
+    background-color: #1c1b1e;
+
+    .container {
+        max-width: 1170px;
+        margin: 0 auto;
+        padding: 0 20px;
+    }
+
+    &__wrapper {
+        text-align: center;
+    }
+
+    &__title {
+        color: white;
+        font-size: 30px;
+        padding: 50px 0;
+        font-family: "Dancing Script", cursive;
+    }
+
+    &__content {
+        color: white;
+        font-family: 'Open Sans', sans-serif;
+        font-size: 20px;
+        padding: 30px 0;
+        word-spacing: 10px;
+    }
+}
+
+.specificity {
+    background-color: #1c1b1e;
+
+    .container {
+        max-width: 1170px;
+        margin: 0 auto;
+        padding: 50px 0;
+    }
+
+    &__wrapper {
+        display: flex;
+        justify-content: space-between;
+        align-items: center;
+    }
+
+    &__content {
+        max-width: 40%;
+        text-align: center;
+        padding-right: 20px;
+    }
+    
+    &__title {
+        color: white;
+        font-size: 20px;
+
+        h3 {
+            font-size: 35px;
+            font-family: "Dancing Script", cursive;
+        }
+
+        h2 {
+            font-size: 35px;
+            padding: 10px 0;
+            font-family: "Dancing Script", cursive;
+        }
+    }
+
+    &__text {
+        color: white;
+        font-family: 'Open Sans', sans-serif;
```

Comment:
you should add most repetitive font-famity in body and delete it everywhere(also in common styles file)
I see that it's font-family: 'Open Sans', sans-serif;